tests/x11/gnomecase/change_password.pm

```diff
@@ -61,7 +62,18 @@ sub reboot_system {
 sub switch_user {
     assert_and_click "system-indicator";
     assert_and_click "user-logout-sector";
-    assert_and_click "switch-user";
+    assert_screen([qw(switch-user logout-user)]);
+    if (match_has_tag('switch-user'))
+    {
```

Comment:
I am surprised that the tidy check even accepted this. Please use the following bracketing style:

```perl
if (<condition>) {
…
}
else {
…
}
```